heron/instance/tests/java/com/twitter/heron/network/HandleReadTest.java

```diff
@@ -251,7 +145,7 @@ private Message constructMockMessage() {
       rootId.setTaskid(0);
       dataTuple.addRoots(rootId);
 
-      String s = "";
+      String s;
```

Comment:
a nitpick, we can replace this and following `if..else` with `String s = ((i&1) == 0) ? "A" : "B";`